app/views/candidate_mailer/deferred_offer_reminder.text.erb

```diff
@@ -0,0 +1,15 @@
+Dear <%= @application_form.first_name %>,
+
+# Reminder of your deferred offer
+
+On <%= @application_choice.offer_deferred_at.to_s(:govuk_date) %>, <%= @course_option.course.provider.name %> deferred your offer to study <%= @course_option.course.name_and_code %>.
+
+You'll shortly receive another email from us, confirming that you’ll start training in the new academic year (<%= RecruitmentCycle.current_year %>). If you don’t receive this message, you should contact us at [becomingateacher@digital.education.gov.uk](mailto:becomingateacher@digital.education.gov.uk).
```

Comment:
In order for training to commence, the provider must first reinstate the offer, confirm the conditions of the offer are met (which can happen at the same time as reinstating the offer, but this is less likely) and only then is the candidate 'recruited'. When their training actually starts depends on the course etc. and we won't be emailing them about that. This is the provider's responsibility.

Considering all this, this sentence appears a little inaccurate. Any suggestions @adamsilver ?

Reply:
Yes that's a good point. How about changing this line to something like:
`You'll shortly receive another email from us, confirming the offer. If you don’t receive this message, you should contact {{Provider}}`
This makes it clear that the candidate should expect an email confirming (reinstating) their offer, and it they don't get one they should chase the provider.
What do you think @adamsilver ?

Reply:
I'm thinking:

`You’ll shortly receive an email to confirm the offer. If you do not receive this, you should contact {{provider}}`